server/views/pages/activePunishments.njk

```diff
@@ -0,0 +1,38 @@
+{% extends "./index.njk" %}
+{% from "../macros/conditionallyShow.njk" import conditionallyShow %}
+{% block body %}
+    <div class="govuk-grid-row">
+        <div class="govuk-grid-column-three-quarters">
+            <h1>{{name}}’ active punishments</h1>
```

Comment:
Just adding the apostrophe only works for names ending in "S"
Needs a new filter to add either `'` or `'s` depending on the last letter of the surname.
Or maybe a new `option` on `formatName`?